hyperspy/gui/egerton_quantification.py

```diff
@@ -145,7 +145,8 @@ def bg_to_plot(self, axes_manager=None, fill_with=np.nan):
         return to_return
 
     def span_selector_changed(self):
-        if (self.ss_left_value is np.nan) or (self.ss_right_value is np.nan):
+        if self.ss_left_value is np.nan or self.ss_right_value is np.nan or\
+                self.ss_right_value <= self.ss_left_value:
```

Comment:
I see, but given that I don't remember why I put it there we can:

i) remove it and hope for the best
ii) Leave it where it is

If it is not broken don't touch it, so I would go for ii.